docs/changelog.md

```diff
@@ -1,7 +1,8 @@
 # TBD
+### Fixes
+* An error in this document
```

Comment:
I generally prefer to start these sorta outcome-like, with a verb - e.g. "Fixed an error in this document"

I know there's the `Fixes` section just above (so I'm not _so_ hardline on this), but just tend to think it's a little nicer like:
```
### Fixes
* Fixed an error in this document
* Patched a bug where iproute containers weren't getting torn down
```
than
```
### Fixes
* An error in this document
* A bug where iproute containers weren't getting torn down
```
Just my feelings though; what do you think?

Reply:
I agree to add the Fixed word at the beginning of the sentence